native/src/modules/app/components/App.js

```diff
@@ -56,26 +58,28 @@ const App = () => {
 
   return (
     <Provider store={store}>
-      <StaticServerProvider>
-        <I18nProvider>
-          <SafeAreaProvider>
-            <>
-              <StatusBarContainer />
-              <IOSSafeAreaView>
-                <NavigationContainer onStateChange={onStateChange} linking={linking}>
-                  <NavigatorContainer
-                    routeKey={routeKey}
-                    routeName={routeName}
-                    languageCode={languageCode}
-                    cityCode={cityCode}
-                  />
-                </NavigationContainer>
-              </IOSSafeAreaView>
-              {routeName && <PermissionSnackbarContainer routeName={routeName} />}
-            </>
-          </SafeAreaProvider>
-        </I18nProvider>
-      </StaticServerProvider>
+      <ThemeProvider theme={buildConfig().lightTheme}>
```

Comment:
This allows us to use hooks to get the theme and write StyledComponents without having to pass the theme manually, like it is in web already.